### PR TITLE
Rename Glue data type

### PR DESCRIPTION
**Pull request by tksfjt1024**
Athena doesn't support uppercase letters and spaces.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 group = "pro.civitaspo"
-version = "0.5.0"
+version = "0.5.1"
```
Comment (civitaspo) [inline build.gradle]: I do this change when releasing, so could you remove this change?

**Review by civitaspo — CHANGES_REQUESTED** [review]
Thanks for the PR! There is nothing wrong with this change. Could you revert only the version-related changes?

File: build.gradle (hunk 1 above)
Comment (tksfjt1024) [reply build.gradle]: Reverted. Please confirm

**Review by civitaspo — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (civitaspo) [reply build.gradle]: thx

**Comment by civitaspo** [comment]
I do not know why GHA does not run, but I checked the tests for your PR locally, so ship it.

**Comment by civitaspo** [comment]
Released https://github.com/civitaspo/embulk-output-s3_parquet/releases/tag/0.5.1

**Comment by tksfjt1024** [comment]
Thanks

**Pull request merged**